migrations/README.md

```diff
@@ -0,0 +1,33 @@
+Creating a new migration:
+-------------------------
+
+To create a new migration file (e.g. after updating your models), follow these steps:
+1. Make sure that your database is created and up to date, by running: 
+
+```
+$ docker-compose run lando-api db upgrade
+```
+
+4. Generate a new revision by running 
```

Comment:
Docs, very nice!

Guessing there were more steps before hand, so this should now be 2, then 3 etc.

Reply:
Heh. Yes indeed, will fix.